packages/design-system-docs/src/pages/components/Table/Table.example.jsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableDataCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from '@cmsgov/design-system';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import uniqueId from 'lodash.uniqueid';
+
+const simpleHeaders = [
+  { title: 'Branch', key: 'branch', width: '10' },
```

Comment:
I would also personally not define `width` for all of these, maybe 1 only. Generally speaking I think we want users to not use width unless they really need to right? The default table spacing should handle it most of the time